vnext/Scripts/Tfs/Layout-Headers.ps1

```diff
@@ -11,6 +11,7 @@ param(
 
 [xml]$props = gc $PSScriptRoot\..\..\Directory.Build.props
 [string] $FollyVersion = $props.Project.PropertyGroup.FollyVersion;
```

Comment:
Just noticed this change, but if the versions fully comply with the [uint16].[uint16].[uint16].[uint16] pattern, we should use the `[version]` type instead.

Reply:
Never mind. Their versioning, for reasons I don't understand, allows unorthodox usage of integers (i.e. `00`).